src/PockyBot.NET/Services/Triggers/Status.cs

```diff
@@ -115,7 +115,7 @@ private static string GetPenaltyPegsReceivedText(List<Persistence.Models.Peg> pe
 
         private static string FormatPeg(Persistence.Models.Peg peg)
         {
-            return $"* **{peg.Receiver.Username}** — \"_{peg.Comment}_\"";
+            return $"* **{peg.Receiver.Username}** — \"_{peg.Comment.Replace("\n", " / ")}_\"";
```

Comment:
Should this also replace `\n\n` with `.//.` instead of `./../.`? (using dots in place of spaces because GitHub doesn't seem to like spaces in these locations)